receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -50,32 +52,46 @@ const (
 	TimingTypeName    TypeName = "timing"
 	TimingAltTypeName TypeName = "timer"
 
-	GaugeObserver   ObserverType = "gauge"
-	SummaryObserver ObserverType = "summary"
-	DisableObserver ObserverType = "disabled"
-
-	DefaultObserverType = DisableObserver
```

Comment:
We should not remove this: this is part of the public API and would be a breaking change to remove it. Maybe the `protocol` folder should be `internal` to the statsdreceiver module, but that should be a different PR.

Reply:
Will do.